toplevel/coqloop.ml

```diff
@@ -261,8 +261,8 @@ let rec discard_to_dot () =
     | e when CErrors.noncritical e -> ()
 
 let read_sentence ~state input =
-  let open Vernac.State in
-  try Stm.parse_sentence ~doc:state.doc state.sid input
+  (* XXX: careful with ignoring the state Eugene!*)
```

Comment:
I don't know the expression. Where does it come from?

`XXX` = to be eventually addressed?

Reply:
https://www.youtube.com/watch?v=YtZqNAI4pBk

Reply:
Ah ah, that was my weak spot, I would have merged any PRs with a reference to Pink Floyd. Too late! :)

Reply:
@ejgallego: ah, not the PF I know well.
@maximedenes; Why too late?

Reply:
Oh, just that merge almost only kernel PRs now. And PF won't be enough to justify a kernel change, I'm afraid :)

Reply:
Maybe @gares could this time not disagree with you and accept to let you merge this one as a weak-spot exception :)

Reply:
I hate PF